ur_robot_driver/CHANGELOG.rst

```diff
@@ -1,3 +1,10 @@
+2.3.8 (2024-06-17)
+------------------
+* Remove extra spaces from start_ursim statement in tests ( backport of `#1010 <https://github.com/UniversalRobots/Universal_Robots_ROS2_Driver/issues/1010>`_)
+* Use robot_receive_timeout instead of keepalive_count
```

Comment:
```suggestion
* Use robot_receive_timeout instead of keepalive_count (`#1003 <https://github.com/UniversalRobots/Universal_Robots_ROS2_Driver/issues/1003>`_)
```